opencti-platform/opencti-graphql/src/resolvers/container.js

```diff
@@ -49,8 +49,9 @@ const containerResolvers = {
       fieldPatch: ({ input, commitMessage, references }) => stixDomainObjectEditField(context, context.user, id, input, { commitMessage, references }),
       contextPatch: ({ input }) => stixDomainObjectEditContext(context, context.user, id, input),
       contextClean: () => stixDomainObjectCleanContext(context, context.user, id),
-      relationAdd: ({ input }) => stixDomainObjectAddRelation(context, context.user, id, input),
-      relationDelete: ({ toId, relationship_type: relationshipType }) => stixDomainObjectDeleteRelation(context, context.user, id, toId, relationshipType),
+      relationAdd: ({ input, commitMessage, references }) => stixDomainObjectAddRelation(context, context.user, id, input, { commitMessage, references }),
+      // eslint-disable-next-line max-len
```

Comment:
can you split the line instead ? Or is it really not readable ?

Reply:
I didn't find a nice way to keep it readable and keep lint happy, so i went with this. Also, in other resolvers like stixCoreRelationship, the relationDelete line was also handled this way